pulp_container/app/urls.py

```diff
@@ -7,7 +7,28 @@
     VersionView
 )
 
-router = SimpleRouter(trailing_slash=False)
+
+class RegistryRouter(SimpleRouter):
```

Comment:
@dkliban do i understand correctly that this change will be applied to the all the routes?
i was wondering if applying this regexp directly on the viewset won't be a more isolated change?
```
[ipanova@fluffy pulp_container]$ git diff
diff --git a/pulp_container/app/viewsets.py b/pulp_container/app/viewsets.py
index f78c0ef..cc0db59 100644
--- a/pulp_container/app/viewsets.py
+++ b/pulp_container/app/viewsets.py
@@ -739,7 +739,7 @@ class Manifests(ViewSet):
     """
     ViewSet for intereacting with Manifests
     """
-
+    lookup_value_regex = '[^/^&*%!~@#+=?]+'
     # allow anyone to access
     authentication_classes = []
     permission_classes = []
```

Reply:
good call. i'll make this change.